### PR TITLE
Clean code. Pls test it before pushing

### PR DESCRIPTION
**Pull request by com-master**

### CODE REVIEW

File: src/Data.sol [hunk 1]
```diff
@@ -72,14 +70,14 @@ contract Data is IData, IndexResolver {
     function redeployIndex() public view onlyOwner {
         require (msg.value >= (_indexDeployValue * 2), DataErrors.value_less_than_required);
         tvm.accept();
-        tvm.rawReserve(msg.value, 1);
+
```
Comment (itgoldio) [inline src/Data.sol:73]: Need to use rawReserve
Owner can freez nft if use contract balance

File: src/Data.sol [hunk 2]
```diff
@@ -72,14 +70,14 @@ contract Data is IData, IndexResolver {
...
 
         address oldIndexOwner = resolveIndex(address(0), address(this), _addrOwner);
-        IIndex(oldIndexOwner).destruct();
+        IIndex(oldIndexOwner).destruct(); // Like tvm.commit
         address oldIndexOwnerRoot = resolveIndex(_addrRoot, address(this), _addrOwner);
```
Comment (itgoldio) [inline src/Data.sol:77]: Why did you add this comment, can explain it?

File: src/NftRoot.sol [hunk 3]
```diff
@@ -99,11 +94,6 @@ contract NftRoot is DataResolver, IndexResolver {
         tvm.accept();
         _indexDeployValue = indexDeployValue;
     }
-    
-    function setProcessingValue(uint128 processingValue) public onlyOwner {
-        tvm.accept();
```
Comment (itgoldio) [inline src/NftRoot.sol:104]: it will use in samples

File: src/NftRoot.sol [hunk 4]
```diff
@@ -119,10 +109,6 @@ contract NftRoot is DataResolver, IndexResolver {
         return _indexDeployValue;
     }
 
-    function getProcessingValue() public view returns(uint128) {
-        return _processingValue;
```
Comment (itgoldio) [inline src/NftRoot.sol:123]: it will use in samples

Comment (com-master) [reply src/NftRoot.sol:123]: Developers will add by their self's

File: src/Data.sol (hunk 2 above)
Comment (com-master) [reply src/Data.sol:77]: It's for you. What you don't need their tvm rawReserve

File: src/Data.sol (hunk 1 above)
Comment (com-master) [reply src/Data.sol:73]: You already tvm accept. If you did it smart contract start pay for this message. And you don't need tvm rawReserve

**Comment by itgoldio** [comment]
thx.
It Fixed it in another PR
